installer/src/services/arduino-cli.js

```diff
@@ -48,7 +48,37 @@ const getBoards = ({
     });
 };
 
-const upload = ({ dstPath, fqbn, port, onData, onClose, onError }) => {
+const installRequiredCores = async ({ fqbnList, onData }) => {
```

Comment:
AFAIK this should be a platform list not an fqbn list
```suggestion
const installRequiredCores = async ({ platforms, onData }) => {
```